python/liveconfig3

```diff
@@ -0,0 +1,2 @@
+addr=digtest1@testrun.org mail_pw=diggydug 
+addr=digtest2@testrun.org mail_pw=diggydug
```

Comment:
whoops, guess you're deleting those accounts now.

Also, password managers FTW.